packages/kit-bg/src/vaults/impls/neo/Vault.ts

```
@@ -0,0 +1,542 @@
+    }
+  }
+
+  /** ************************************ DApp Method ************************************* */
+  // Invoke
+  async createInvokeInputs(invokeArgs: IInvokeArguments) {
+    const { args, scriptHash, operation } = invokeArgs;
+    return {
+      scriptHash,
+      operation,
+      args: args.map((item: IArgument) => {
+        if (item && item.type && item.type === 'Address') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)